src/rust/ensogl/src/display/style/path.rs

```diff
@@ -24,37 +24,42 @@ impl Path {
     pub fn from_segments<T,I,Item>(t:T) -> Self
     where T : IntoIterator<IntoIter=I,Item=Item>,
           I : DoubleEndedIterator + Iterator<Item=Item>,
-          Item : Into<String> {
+          Item : ToString {
```

Comment:
Because it now works on `&&str`. Which is super stupid, but that's the only way to allow passing here things like `&["foo","bar"]` - here the iterator gives you `&&str`